scripts/tests/ota_test.sh

```diff
@@ -15,30 +18,28 @@ OTA_REQUESTOR_FOLDER="out/ota_requestor_debug"
 CHIP_TOOL_APP="chip-tool"
 CHIP_TOOL_FOLDER="out"
 
-killall -e "$OTA_PROVIDER_APP" "$OTA_REQUESTOR_APP"
+killall -INT -e "$OTA_PROVIDER_APP" "$OTA_REQUESTOR_APP"
 rm -f "$FIRMWARE_OTA" "$FIRMWARE_BIN" "$OTA_DOWNLOAD_PATH"
 
-set -e
```

Comment:
Any reason for removing this? We won't catch and exit on errors now (like if compilation fails in the next step, the script will continue instead of exiting)

Reply:
I think it may have caused the CI to fail, but I will run a few times with and without to see

Reply:
It is causing CI to fail.